### PR TITLE
Basic page structure

### PR DESCRIPTION
**Pull request by JuanLPalacios**
Project setup and implementation of basic structure, layout, and functions mockup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "leaderboard",
+  "description": "",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --mode development",
+    "build": "webpack --mode production"
+  },
```
Comment (moise10r) [inline package.json]: - [ ]  I try running npm run start after installing dependencies by running `npm install` then `npm run start` but the app is not opened in the browser. I checked your package.json file and found out that the `start` script which is responsible for starting your app in the browser is missing, I added the start script `"start": "webpack serve --open"` and I was able to run your app with `npm run start` but I  got the error below :point_down: 

badweel

- But don't worry to solve this got to your webpack.config.js and add this :point_down: 
`mode: 'production', ` or `mode: 'development',`

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname),
+    clean: true,
+  },
```
Comment (moise10r) [inline webpack.config.js:10]: - [ ] Add this 

```
 module.exports = merge(common, {
    mode: 'production', //add this line here or
   // mode: 'development',
  });
```

- I found it takes longer to run the app on browser as expected but if you can figure out how to fix this that will be great :+1:

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@JuanLPalacios,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Good commit history ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [ ] Your README looks great 👍🏻  but  I think it will be great to add instructions on how to set the project up locally

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -2,4 +2,54 @@
...
+to set it up yo have to first run 
+
+```
+npm install
+```
+
+after that you can run the local development server with:
+```
+npm start
+```
```
Comment (JuanLPalacios) [inline README.md:21]: these are the setup instructions

File: package.json (hunk 1 above)
Comment (JuanLPalacios) [reply package.json]: you are removing the mode flag from the command, to ad the open command  use
```
 "start": "webpack serve --mode development --open",
```

File: webpack.config.js (hunk 2 above)
Comment (JuanLPalacios) [reply webpack.config.js:10]: check the other comment on top, please

**Review by medaminedev66 — APPROVED** [review]
# Hi @JuanLPalacios, 

## Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- As the previous reviewer said, it takes longer to run the app on the browser. If you can figure out how to fix this, that will be great 👍

**You may merge the changes**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Amine Smahi_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**